.github/workflows/docs-checks.yml

```diff
@@ -48,11 +48,21 @@ jobs:
           sudo apt-get install -y cmake pandoc
           pip --version
           # python -m pip install --upgrade --user pip
-          pip install -e . --quiet -r requirements/${{ matrix.pkg }}/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+          pip install -e . -r requirements/${{ matrix.pkg }}/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
           pip install -r requirements/${{ matrix.pkg }}/devel.txt
           pip list
         shell: bash
 
+      - name: Install PyTorch Lightning from source for app docs (hotfix)
+        if: ${{ matrix.pkg == 'app' }}
+        env:
+          PACKAGE_NAME: pytorch
+        run: |
+          git checkout setup.py  # reverts _PAKCAGE_NAME = "app"
+          pip install -e . -U
+          pip list
+        shell: bash
```

Comment:
why it is needed when it is installed in the step above?
https://github.com/Lightning-AI/lightning/pull/14426/files#r956952600

Reply:
Because the installed PL version in the step above is one of the PyPI releases but not the dev one. This new step replaces PL installed via PyPI with the dev version in the branch.

Reply:
Interesting, that could be because of:
https://github.com/Lightning-AI/lightning/blob/7987a1b453150f6c9733db24ab0cdcf11e60dd9d/requirements/app/docs.txt#L5

Reply:
As commented in https://github.com/Lightning-AI/lightning/pull/14426#discussion_r959545594, it didn't work. I have no idea. (Anyone seeing this comment, feel free to push commits directly... which would be very nice...)

---

Or, the way I originally wrote (https://github.com/Lightning-AI/lightning/pull/14426/commits/5be169edc863469d9049b7485d980822cedacd50) works at least, so I was wondering if you'd agree to revert to the commit, i.e. to write the following:
```yaml
      - name: Install PyTorch Lightning from source for app docs (hotfix)
        if: ${{ matrix.pkg == 'app' }}
        env:
          PACKAGE_NAME: pytorch
        run: |
          git checkout setup.py  # reverts _PAKCAGE_NAME = "app"
          pip install -e . -U
          pip list
        shell: bash
```